unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs

```diff
@@ -530,5 +527,12 @@ public static void SetLayerRecursively(Transform transform, int layer)
         /// <param name="volume">Linear volume (0 to 1)</param>
         /// <returns>Value for audio mixer group volume</returns>
         public static float ToAudioMixerGroupVolume(float volume) { return (ToVolumeCurve(volume) * 80f) - 80f; }
+
+        public static string GetHierarchyPath(this Transform transform)
```

Comment:
If this method isn't used we should remove it?

Reply:
Oops I bring this util from the Avatar Refactor for debugging and I forgot to remove it. On it